avocado/plugins/jsonresult.py

```diff
@@ -91,6 +92,27 @@ def render(self, result, job):
                     json_file.write(content)
 
 
+class JSONInit(Init):
+
+    name = 'json'
+    description = "JSON job result plugin initialization"
+
+    def initialize(self):
+        help_msg = ('Enable JSON result format and write it to FILE. '
+                    'Use "-" to redirect to the standard output.')
+        settings.register_option(section='job.run.result.json',
```

Comment:
Hi @clebergnu, just one question/suggestion:

Why we don't take this opportunity and change this to `job.results.json`? Do we really need the `run` section? I mean, we are talking about the results of a job, right? I know that are the "results of a job run" but maybe this is too much information for the user.

Reply:
If I understood correctly, it is keeping the hierarchy defined in the PR description. In this case, a job do not generate JSON.

Reply:
Okay, got it. From my perspective as a user, besides the verbosity, it makes sense to me, but I tend to prefer verbose definitions as I think it avoids ambiguity. Anyway, it is a personal point of view.

Reply:
Yeah, I don't object dropping `run`, but at the same time, I wonder id we'd need other namespaces.  For instance, suppose we have a `job.result` namespace that has to do *not* with running a test, but collecting/uploading/publishing/notifying results? Brainstorming:
 * job.result.collect = namespace about collecting results of tests in the job (not necessarily at run time)
 * job.result.publish = namespace about publishing/uploading job results to a central database or repo
 * job.result.notify = namespace about notifiers such as email or IRC bots about test test results

Again, I proposed what I feel it's more appropriate, but there's a lot of guessing and I don't claim to be right :)

Reply:
I like explicitly when it is related to code, for developers, but for users, I tend to keep it simple. This proposal here makes sense and it is simple for me.

Again it was just a suggestion, brainstorming. Maybe we will see the need soon.

Reply:
OK, so I'll follow up the idea with a non-draft PR. Thanks @beraldoleal and @willianrampazzo